SPECS/openjdk8/openjdk8.spec

```diff
@@ -86,13 +90,13 @@ export CXXFLAGS=$(echo $CXXFLAGS | sed "s/-Wall//" | sed "s/-Wformat// | sed "s/
 unset JAVA_HOME &&
 ./configur* \
 	--with-target-bits=64 \
-	--with-boot-jdk=/usr/lib/jvm/OpenJDK-212-b04-bootstrap \
+	--with-boot-jdk=%{_lib}/jvm/OpenJDK-212-b04-bootstrap \
 	--disable-headful \
-	--with-cacerts-file=/usr/lib/jvm/OpenJDK-212-b04-bootstrap/jre/lib/security/cacerts \
+	--with-cacerts-file=%{_lib}/jvm/OpenJDK-212-b04-bootstrap/jre/lib/security/cacerts \
 	--with-extra-cxxflags="-Wno-error -std=gnu++98 -fno-delete-null-pointer-checks -fno-lifetime-dse" \
 	--with-extra-cflags="-std=gnu++98 -fno-delete-null-pointer-checks -Wno-error -fno-lifetime-dse" \
-	--with-freetype-include=/usr/include/freetype2 \
-	--with-freetype-lib=/usr/lib \
+	--with-freetype-include=%{_includedir}/freetype2 \
+	--with-freetype-lib=%{_lib} \
```

Comment:
Ah, we need to be careful here. The linter should use `%{_libdir}` for `/usr/lib`, **NOT** `%{_lib}`. We're working on fixing that, but for now, please replace all `%{_lib}` with `%{_libdir}`.